### DISCUSSION TITLE
Bug d'affichage

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#2012] [Idées]
Sur la page "Satellites" de la fiche d'une cuisine centrale, le nombre de cantines satellites indiqué dans la phrase "Vous en avez renseigné X" est faux : le nombre indiqué est le nb de CS affiché sur la page, et non le nombre de CS référencées en tout (cf. PJ).

![Capture d’écran 2022-10-18 à 10 43 04](https://user-images.githubusercontent.com/93709421/196382697-c6107c3e-eca7-4e77-9070-313958cdedb2.jpg)

@hfroot @alemangui

**Comment by hfroot** [comment]
Ah bizarre, merci. 

Pour les bugs tu peux créer des issus directe. Je le crée maintenant :)

**Reply by ghost** [reply]
OK, je pensais qu'il fallait que je crée une discussion avec l'étiquette "bug". La prochaine fois, je créerai directement l'issue. Merci, @hfroot.
